components/src/instrument/PipetteSelect.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 import * as React from 'react'
 import groupBy from 'lodash/groupBy'
-
+import { Flex } from '@opentrons/components'
```

Comment:
Since you're already in components here, you may want to do a relative import, instead. I've noticed this sort of thing confuse webpack before

```suggestion
import { Flex } from '../primitives'
```